### PR TITLE
Add link to Scio (Scala) interface.

### PR DESCRIPTION
**Pull request by robertwb**

### CODE REVIEW

**Comment by robertwb** [comment]
R: @nevillelyh, @melap

**Comment by nevillelyh** [comment]
LGTM thanks!

File: src/get-started/beam-overview.md [hunk 1]
```diff
@@ -23,6 +23,8 @@ Beam currently supports the following language-specific SDKs:
 * Java ![Java logo]({{ "/images/logos/sdks/java.png" | prepend: site.baseurl }})
 * Python ![Python logo]({{ "/images/logos/sdks/python.png" | prepend: site.baseurl }})
 
+A Scala ![Scalal logo]({{ "/images/logos/sdks/scala.png" | prepend: site.baseurl }}) interface is also available as [Scio](https://github.com/spotify/scio).
```
Comment (melap) [inline src/get-started/beam-overview.md]: typo (Scalal) - though it won't be visible except in alt text

Comment (robertwb) [reply src/get-started/beam-overview.md]: Thanks. Fixed.

**Comment by melap** [comment]
Generated it locally - the builds seem to be slow today, the image ends up being considerably larger than the others. could resize the image, or alternately just specify a height onto the end in the page? can just cut and paste this:
Scala logo{: height="45px"}

**Comment by robertwb** [comment]
Yes, added a height of 45px as suggested.

**Comment by melap** [comment]
@asfgit merge

**Comment by melap** [comment]
Yeah, image needs manual add. 😞  I'll open a PR first thing tomorrow morning.

**Pull request closed without merging**